docs-generation/tests/test_translation_fixes.py

```
@@ -0,0 +1,197 @@
+            nodes, concurrency=4, process_fn=lambda n: "success"
+        )
+        self.assertFalse(aborted)
+        self.assertEqual(sorted(results["success"]), sorted(nodes))
+        self.assertEqual(results["failed"], [])
+
+    def test_circuit_breaker_trips_on_consecutive_failures(self):
+        nodes = [f"Node{i}" for i in range(20)]
+        results, aborted = btd.translate_nodes_concurrently(
+            nodes,
+            concurrency=1,  # deterministic ordering for the breaker test
+            process_fn=lambda n: "failed",
```

Ruff (0.16.1)

[warning] Unused lambda argument: `n`

(ARG005)